### ISSUE TITLE
[Accessibility] Search results page has no "Skip Links" (WCAG 2.0, Success Criteria 2.4.1)

### ISSUE THREAD

**Issue by spudthebud** [DOAJ/doaj#2168]
### Overview
DOAJ search results page (for searching journals) does not contain "By Pass Block" as required by WCAG 2.0 [success criteria 2.4.1](https://www.w3.org/WAI/WCAG21/Understanding/bypass-blocks.html)

### User story (fictitious)
I am unable to use a mouse due to limited dexterity in my hands. I rely on navigating by the keyboard. I press tab or shift tab to move the keyboard focus.  

I have done a journal search and would like to apply a filter. There is no "skip to facets" bypass block that will take me to the facets.  This means I am going to have to press tab lots!

###Environment
Windows 11
Chrome 106

MyID=252.

**Comment by dommitchell** [comment]
Thank you for reporting this. We have duplicated the bug to our bug repo and I will update here once your ticket has been considered.

**Comment by dommitchell** [comment]
Hi Mark, @spudthebud 

I have some good news. We released a change to our site yesterday that fixed this. You can see it live on our site: https://doaj.org/ If you are happy to, please close this issue when ready.

You raised another two issues #2167 and #2169 and we will be releasing those shortly. I'll update each issue when that happens.
